16-google/cms.ts

```diff
@@ -0,0 +1,28 @@
+import { Event, normal, metronome, TimedDependency, Timeout } from "../../src";
+
+export class ClusterManagementSoftware extends TimedDependency {
```

Comment:
No need to extend TimedDependency since you don't use any of the functionality you are inheriting. Just extend Stage.

Also, rename the file to be cluster-manangement-software.ts or change the class name to be CMS, so that the file relates to the class it holds.